Resources/Prototypes/Tiles/floors.yml

```diff
@@ -32,6 +36,8 @@
   id: floor_white
   name: white floor
   texture: "white"
+  variants: 4
+  placementVariants: [0, 1, 2, 3]
```

Comment:
whats the point of `placementVariants` exactly, why cant it just be generated from the # of variants

Reply:
> whats the point of `placementVariants` exactly, why cant it just be generated from the # of variants

tl;dr variants may not always be for just simple mapping visuals so I thought i'd plan ahead.

Reply:
make sense